app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -526,6 +531,17 @@ public void onFsViewerConfig(FilesystemViewerData.Options dopt) {
                 }
                 return true;
             }
+            case R.id.action_set_font_size: {
+                final DisplayMetrics displayMetrics = getContext().getResources().getDisplayMetrics();
+                final int oldSize = (int) (_hlEditor.getTextSize() / displayMetrics.scaledDensity);
+                SearchOrCustomTextDialogCreator.showFontSizeDialog(getActivity(), oldSize, (v) -> {
+                    final int newSize = Integer.parseInt(v);
+                    if (newSize != oldSize) {
+                        _hlEditor.setTextSize(TypedValue.COMPLEX_UNIT_SP, (float) newSize);
+                    }
+                    _appSettings.setDocumentFontSize(getPath(), newSize);
```

Comment:
Setting saved only if user explicitly sets font size.

Reply:
I don't think we need the displayMetrics & oldSize. Just 

```
callback {
  editor.setTextSize  
  setDocumentFontsize
}
```

should be fine? And in case user presses cancel (thus opposite of "explicitley set"), nothing happens as the callback not invoked.

whats your view/thought?

Reply:
I have removed the unnecessary test. The displayMetrics and oldSize are used to highlight the current font size. I think that is useful to have

Reply:
Ha! Didn't think of that. Yes, that will work.

Reply:
I have pushed this change